app/code/Magento/CatalogImportExport/Model/Import/Uploader.php

```diff
@@ -197,11 +197,14 @@ public function move($fileName, $renameFileOff = false)
         }
 
         $this->_setUploadFile($tmpFilePath);
-        $rootDirectory = $this->getTargetDirectory()->getDirectoryRead(DirectoryList::ROOT);
+        $rootDirectory = $this->targetDirectory->getDirectoryRead(DirectoryList::ROOT);
         $destDir = $rootDirectory->getAbsolutePath($this->getDestDir());
         $result = $this->save($destDir);
-        unset($result['path']);
-        $result['name'] = self::getCorrectFileName($result['name']);
+
+        if ($result) {
+            unset($result['path']);
+            $result['name'] = self::getCorrectFileName($result['name']);
+        }
 
         // Directory and filename must be no more than 255 characters in length
         if (strlen($result['file']) > $this->maxFilenameLength) {
```

Comment:
with $result = false following call will trigger warning `strlen($result['file']` : `Trying to access array offset on value of type bool`

Reply:
I changed the fix